### PR TITLE
fix #8650 bug(nimbus): use computed enrollment end date on landing page

### LINKED ISSUE #8650: Landing page shows wrong enrollment end date
Here we have an experiment

https://experimenter.services.mozilla.com/nimbus/firefox-android-default-browser-message-test/summary

that enrolled from March 14-March 28 but the landing page shows it enrolled from March 14-March 20

Let's make sure the landing page is using the correct computed enrollment end date.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3366)

### PR DESCRIPTION
**Pull request by jaredlockhart**
Because

* The frontend was using an incorrect calculation to display the enrollment period on the landing page table
* There is a property to compute the correct enrollment end date but it wasn't being used

This commit

* Updates the frontend to use the computed enrollment end date property

### CODE REVIEW

File: experimenter/experimenter/experiments/tests/api/v5/test_queries.py [hunk 1]
```diff
@@ -736,9 +742,10 @@ def test_get_experiment_by_slug_returns_experiment_data(self, lifecycle):
                     }
 
                     startDate
+                    computedDurationDays
```
Comment (eliserichards) [inline experimenter/experimenter/experiments/tests/api/v5/test_queries.py:745]: Why is this added too?

File: experimenter/experimenter/nimbus-ui/src/gql/experiments.ts [hunk 2]
```diff
@@ -131,9 +131,10 @@ export const GET_EXPERIMENT_QUERY = gql`
       }
 
       startDate
+      computedDurationDays
```
Comment (eliserichards) [inline experimenter/experimenter/nimbus-ui/src/gql/experiments.ts:134]: same q here

File: experimenter/experimenter/nimbus-ui/src/types/getExperiment.ts [hunk 3]
```diff
@@ -180,9 +180,10 @@ export interface getExperiment_experimentBySlug {
   proposedDuration: number;
   readyForReview: getExperiment_experimentBySlug_readyForReview | null;
   startDate: DateTime | null;
+  computedDurationDays: number | null;
```
Comment (eliserichards) [inline experimenter/experimenter/nimbus-ui/src/types/getExperiment.ts:183]: same q here

**Review by eliserichards — COMMENTED** [review]
Lookin' good! One question for you:

**Review by yashikakhurana — APPROVED** [review]
💪
